flight/Modules/PathPlanner/pathplanner.c

```diff
@@ -253,6 +250,7 @@ static void holdCurrentPosition()
 	pathDesired.StartingVelocity = 5; // This will be the max velocity it uses to try and hold
 	pathDesired.EndingVelocity = 5;
 	pathDesired.ModeParameters = 0;
+	pathDesired.Waypoint = -1;
```

Comment:
just to confirm my understanding, this is a sanity check and should not be relevant in normal usage.

regardless it should be duplicated in holdLastPosition